packages/api/lambdas/sf-event-sqs-to-db-records/index.js

```diff
@@ -131,9 +133,8 @@ const handler = async (event) => {
 
   return await Promise.all(sqsMessages.map(async (message) => {
     const executionEvent = parseSQSMessageBody(message);
-    const cumulusMessage = await getCumulusMessageFromExecutionEvent(executionEvent);
-
     try {
+      const cumulusMessage = await getCumulusMessageFromExecutionEvent(executionEvent);
```

Comment:
This is an interesting change.... 

Before this change, the lambda would *throw* which..... would result in redrive policy retrying this up to 10 times, and then put the message in the DLQ. 

Is the intention here to limit those retries in that way?  If so, why?  

```
const cumulusMessage = await getCumulusMessageFromExecutionEvent(executionEvent);
```